beakerx_base/beakerx_base/utils.py

```diff
@@ -199,6 +199,8 @@ def default(self, obj):
             return self.default(obj.tolist())
         elif isinstance(obj, np.int64):
             return self.default(obj.item())
+        elif isinstance(obj, np.bool_):
```

Comment:
@LeeTZ @jaroslawmalekcodete I wonder if it's not worth to add case for https://docs.scipy.org/doc/numpy-1.15.0/reference/generated/numpy.generic.html to cover future numpy type casts?

Reply:
It's good for now